packages/spacecat-shared-rum-api-client/src/common/traffic.js

```diff
@@ -174,9 +219,11 @@ export function classifyTrafficSource(url, referrer, utmSource, utmMedium, track
     && rule.utmMedium(sanitize(utmMedium))
     && rule.tracking(trackingParams)
   ));
+  const channel = classifyReferrer(referrerDomain);
```

Comment:
hint for the `vendor` can exist not only in referrer, but also in utm_source, or even in utm_medium. We should consider all those, in respective order: referrer, utm_source, utm_medium.